src/app/workflow/workflow.component.html

```diff
@@ -47,6 +47,26 @@ <h2 class="text-center">The workflow linked to could not be found!</h2>
       This {{ workflow.entryTypeMetadata.term }} was archived on {{ workflow.dbUpdateDate | date: 'yyyy-MM-dd' }}. It is now read-only.
     </mat-card>
   </div>
+  <div class="row m-1 mt-3 mb-0" *ngIf="!isWorkflowPublic && workflow.mode === 'DOCKSTORE_YML' && gitHubAppInstalled === false">
```

Comment:
`isWorkflowPublic` is a confusing name. I was say it was wrong, but you're using it consistently with how it's already being used.

My expectation: `isWorkflowPublic` -> whether the workflow is public (published or not)
Actuality: `isWorkflowPublic` -> whether the page is public or not (public view or My workflows).

No need to change, I was just commenting because I initially thought you wouldn't be showing this warning for non-public workflows.